lib/Basics/ReadWriteLock.cpp

```diff
@@ -18,115 +18,86 @@
 ///
 /// Copyright holder is ArangoDB GmbH, Cologne, Germany
 ///
-/// @author Frank Celler
-/// @author Achim Brandt
+/// @author Max Neunhoeffer
 ////////////////////////////////////////////////////////////////////////////////
 
 #include "ReadWriteLock.h"
 
-#if 0
 using namespace arangodb::basics;
 
-////////////////////////////////////////////////////////////////////////////////
```

Comment:
ehhh...can was this an accident? it is completely unrelated

Reply:
It was intentional. 
We had an unused old R/W lock implementation with deactivated code. All that was remaining was just  `typedef ReadWriteLockCPP11 ReadWriteLock;`. So it can go away safely. I could split that into a separate PR if that helps, but it will take me 30 extra minutes to compile everything again.